package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "svelte-jester",
```

Comment:
When I run `npm install` it removes this line. We probably should try to avoid switching it back and forth every commit. I wonder why it's added for you and removed for me. It doesn't seem like we need it since it's on line 2. I'm running npm 7.17.0. Maybe you're on an older version?

Reply:
nevermind. mine is adding it too